index.html

```diff
@@ -357,10 +394,10 @@ <h2>Contact me</h2>
                 <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required />
               </li>
               <li class="text-area-li">
-                <textarea name="msg" id="msg" maxlength="500" required>Write your messages here</textarea>
+                <textarea name="msg" id="msg" maxlength="500" required placeholder="Write your messages here"></textarea>
               </li>
               <li class="submit-li">
-                <button type="submit" id="getInTouch">Get it Touch</button>
+                <button type="submit" id="getInTouch">Get in touch</button>
```

Comment:
- [ ] Kindly remove the black border around the `Get in touch` button as suggested by the previous reviewer. 
![image](https://user-images.githubusercontent.com/44057863/179302218-20b0c8b2-a5b1-4fb0-aa5d-9d95d18f2702.png)
You can add the `border: none;` rule in `.message-form ul li button` selector in the mobile version so that it will be applied to all screen sizes.